index.html

```diff
@@ -544,7 +551,7 @@ <h2>Test Vectors</h2>
             "@context": [
               "https://www.w3.org/2018/credentials/v1",
               "https://www.w3.org/2018/credentials/examples/v1",
-              "https://example.com/credentials/latest"
+              "https://w3c-ccg.github.io/lds-ed25519-2020/contexts/lds-ed25519-2020-v1.json"
```

Comment:
```suggestion
              "https://w3id.org/security/suites/ed25519-2020/v1"
```

Reply:
https://w3id.org/security/suites/ed25519-2020/v1 dereferences to https://digitalbazaar.github.io/ed25519-signature-2020-context/contexts/ed25519-signature-2020-v1.jsonld which is in different repo from this one and a different context file than this repo has. Is that correct/expected?

Reply:
For the moment, yes, that's correct and expected. 

Ideally, it would point to the context in this repo (but this repo has been out of date for a while). Once we ensure that we have a solid 2020 implementation, we should make sure everyone is fine w/ that 2020 context, move it over to this repo, and then change the w3id.org redirect.

Reply:
Sounds good. Updated in 04c431c